system/Autoloader/FileLocator.php

```diff
@@ -195,7 +196,7 @@ public function search(string $path, string $ext = 'php', bool $prioritizeApp =
         foreach ($this->getNamespaces() as $namespace) {
             if (isset($namespace['path']) && is_file($namespace['path'] . $path)) {
                 $fullPath = $namespace['path'] . $path;
-                $fullPath = realpath($fullPath) ?: $fullPath;
+                $fullPath = realpath($fullPath) !== false ? realpath($fullPath) : $fullPath;
```

Comment:
I don't like that we call `realpath()` twice. In theory, this is cached, but still... I would prefer a separate variable here, especially since we call it in a loop. This happens a few times in a `FileLocator.php`.

```suggestion
                $resolvedPath = realpath($fullPath);
                $fullPath     = $resolvedPath !== false ? $resolvedPath : $fullPath;
```

Reply:
I've been thinking about it too. I chose the short way. I'll fix it